app/templates/index.js

```diff
@@ -1,3 +1,6 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
```

Comment:
Feathers core already includes this. We may only have to add it in the plugin tests.